pkg/envoy/eds/cluster_load_assignment.go

```diff
@@ -53,7 +56,27 @@ func newClusterLoadAssignment(svc service.MeshService, serviceEndpoints []endpoi
 				Value: uint32(lbWeightPerEndpoint),
 			},
 		}
-		cla.Endpoints[0].LbEndpoints = append(cla.Endpoints[0].LbEndpoints, lbEpt)
+
+		// endpoint with Weight set is a remote cluster endpoint.
```

Comment:
Since this comment is associated with the code handling local cluster endpoints, I'd invert it.
```suggestion
		// endpoint without a weight set implies it belongs to the local cluster
```

Reply:
Thanks. Resolved.